### ISSUE TITLE
Installation broken (requirements.txt missing)

### ISSUE THREAD

**Issue by huyz** [senpay/slacker#8]
This is a continuation of the [circular dependency issue](https://github.com/senpay/slacker/commit/ef123eeca57c79b1e83daa39f4f81ef3906d3cd6#r41690608) I brought up and which was [partially fixed](https://github.com/senpay/slacker/commit/cb6112f0a1da17ed5bee30f0f58b291dbdeb7750).

Now there's a need for a `requirements.txt` file, which doesn't seem to be part of the distribution.

**Comment by senpay** [comment]
@huyz Hello, I will have a look

**Comment by jantman** [comment]
Yeah, we were trying to use this package, and the pip installation is just completely broken. After solving the dependency issue with requests (install is completely broken), it tries to use a requirements file that's not present in the package (and shouldn't be used in packages).

**Comment by senpay** [comment]
Sorry for taking that long with feedback. I think I can see the issue: 
![image](https://user-images.githubusercontent.com/12186301/182971794-4b41d19f-d70e-4fd3-8d52-354cb2ce2943.png)

@jantman I am not sure I understand why `requirements.txt` should not be used in packages, can you please elaborate?

**Comment by senpay** [comment]
@jantman @huyz thanks for bringing up this issue - installation is fixed now!

**Comment by huyz** [comment]
@senpay feel free to close. It's been too long, so I can't test right now
